--- origsrc/coq-8.3pl2/lib/system.ml	2010-12-24 03:55:54.000000000 -0600
+++ src/coq-8.3pl2/lib/system.ml	2011-12-17 23:15:38.644364900 -0600
@@ -300,7 +300,7 @@ let run_command converter f c =
   done;
   (Unix.close_process_full (cin,cout,cerr),  Buffer.contents result)
 
-let path_separator = if Sys.os_type = "Unix" then ':' else ';'
+let path_separator = if Sys.os_type = "Win32" then ';' else ':'
 
 let search_exe_in_path exe =
   try
