--- origsrc/coq-8.3pl2/lib/system.ml	2010-12-24 03:55:54.000000000 -0600
+++ src/coq-8.3pl2/lib/system.ml	2011-12-17 23:15:38.644364900 -0600
@@ -300,7 +300,7 @@ let run_command converter f c =
   done;
   (Unix.close_process_full (cin,cout,cerr),  Buffer.contents result)
 
-let path_separator = if Sys.os_type = "Unix" then ':' else ';'
+let path_separator = if Sys.os_type = "Win32" then ';' else ':'
 
 let search_exe_in_path exe =
   try
--- origsrc/coq-8.3pl2/tools/coq_tex.ml4	2010-07-24 10:57:30.000000000 -0500
+++ src/coq-8.3pl2/tools/coq_tex.ml4	2011-12-17 23:16:42.952043100 -0600
@@ -17,12 +17,12 @@
 
 let _ =
   match Sys.os_type with
-    | "Unix" -> ()
-    | _ -> begin
+    | "Win32" -> begin
       	print_string "This program only runs under Unix !\n";
       	flush stdout;
       	exit 1
       end
+    | _ -> ()
 
 let linelen = ref 72
 let output = ref ""
